### ISSUE TITLE
Start building Mac OS Python wheels for yari-py

### ISSUE THREAD

**Issue by MatejKastak** [avast/yari#14]
Currently we support only Linux and Windows platforms.

We should find a way to also build universal MacOS YARI wheels.

- [ ] The first step should be to mention our current limitations in README
- [ ] Then find and document a way to build it on MacOS
- [ ] Optionally prepare a GithubActions workflow for that

**Comment by ITAYC0HEN** [comment]
debugger doesn't work on VSCODE on macos, is it related?

**Comment by MatejKastak** [comment]
Yes it is related, looks like I underestimated how many people run macos. Unfortunately, I don't have a macos available and very little experience with those systems.

We have two options here:
1. Make it somehow build on MacOS in GithubActions.
2. Enable source distribution. This will require some tweaks to the upstream YARA, I have a small patch that exposes necessary functions from libyara we are using in YARI ([ref](https://github.com/MatejKastak/yara/commit/38298ca73bbd2fcca495dc5015adb7675a8e5735)). We will need to move that to upstream, which I have in backlog for the upcomming weeks. After that we will be able to link with a system libyara (expecting users to have YARA installed). But the source python distribution should be able to pick this library and link against that.

The first option is better in my opinion, I will start thinking about it.
